pkg/csi/cinder/nodeserver.go

```diff
@@ -474,8 +475,22 @@ func (ns *nodeServer) NodeGetCapabilities(ctx context.Context, req *csi.NodeGetC
 	}, nil
 }
 
-func (ns *nodeServer) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVolumeStatsRequest) (*csi.NodeGetVolumeStatsResponse, error) {
-	return nil, status.Error(codes.Unimplemented, fmt.Sprintf("NodeGetVolumeStats is not yet implemented"))
+func (ns *nodeServer) NodeGetVolumeStats(_ context.Context, req *csi.NodeGetVolumeStatsRequest) (*csi.NodeGetVolumeStatsResponse, error) {
+	klog.V(4).Infof("NodeExpandVolume: called with args %+v", *req)
+
+	volumePath := req.GetVolumePath()
```

Comment:
This function should comply with csi spec on required fields etc.
please run `make test-cinder-csi-sanity` and fix the issues respective to this function.

Reply:
All sanity test errors related to `NodeGetVolumeStats` are fixed. Some tests fail on other methods, but it seems to be out of the scope of this PR.

Reply:
+1, Awesome!! Thanks.